.azure-pipelines/ultimate-pipeline.yml

```diff
@@ -2307,7 +2408,6 @@ stages:
         parameters:
           masterCommitId: $(masterCommitId)
       - template: steps/install-latest-dotnet-sdk.yml
-      - template: steps/restore-working-directory.yml
```

Comment:
Was it wanted and is it the reason why the coverage job is failing in the pipeline? At first glance I would have said it's not needed but 🤷‍♂️

Reply:
The coverage pipeline was failing because it was running out of drive space. I removed this step (as it shouldn't be needed) to try and fix it.

It's weird tbh, AFACIT, nothing's changed in this PR that should affect the coverage stage. But 🤷‍♂️ I'm inclined to remove it and make "fixing it properly" someone else's problem 😉